src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -1166,12 +1162,18 @@ public boolean syncDatabaseWithRepo(@ValidateStringParam(name = "site") String s
                             objectStateService.insertNewEntry(site, repoOperation.getPath());
                         } else {
                             logger.debug("Set item state for site: " + site + " path: " + repoOperation.getPath());
+                            if (StringUtils.equalsIgnoreCase(state.getPath(), repoOperation.getPath()) &&
+                                    !StringUtils.equals(state.getPath(), repoOperation.getPath())) {
```

Comment:
Add a comment to explain the logic here. My read is: the source is equal to the destination if and only if case is ignored.

Reply:
Another way to say it:

// If they're equal with mixed case, then update ...